lib/caerbannog/queue.rb

```diff
@@ -8,8 +8,9 @@ def self.push(name, payload)
       @message_class.create!(:name => name, :payload => JSON.generate(payload))
     end
 
-    def self.rabbitmq
-      Bunny.run ENV['RABBIT_URL'] do |conn|
+    def self.rabbitmq(rabbit_url = nil)
+      rabbit_url ||= ENV['RABBIT_URL']
```

Comment:
The configuration has always been through methods on the Caerbannog module. The commit you are referring to has never existed.